build-tools/github/validate-e2e

```diff
@@ -37,7 +37,7 @@ if ! $up; then
 fi
 
 # Finally!
-if ! python e2e_tests.py --endpoint http://localhost:$PORT; then
+if ! pytest -q tests_e2e.py; then
```

Comment:
How do we pass in the endpoint now? It would still be beneficial to be able to run these tests against alpha, for example.

If nothing else, we can use an environment variable.